lib/mix/lib/mix/dep/loader.ex

```diff
@@ -304,8 +304,8 @@ defmodule Mix.Dep.Loader do
     |> elem(0)
   end
 
-  defp rebar_children(root_config, manager) do
-    from = Path.absname("rebar.config")
+  defp rebar_children(root_config, manager, dest) do
+    from = Path.join(dest, "rebar.config") |> Path.absname
```

Comment:
We're not using pipes when they are the only *function call*:

```elixir
# We're not doing this:
dest |> Path.join("rebar.config")
```

What @ericmj is doing here is perfectly fine IMO, especially because writing `Path.absname(Path.join(dest, "rebar.config"))` looks not great (IMO ofc) and is one of the reasons why we have the pipe in the first place.